packages/components/src/palette-edit/index.js

```diff
@@ -313,6 +313,7 @@ export default function PaletteEdit( {
 	colors = EMPTY_ARRAY,
 	onChange,
 	paletteLabel,
+	paletteLabelLevel = 2,
```

Comment:
I don't think that this prop name conveys its meaning, since "level" only makes sense in the context of a "heading".

I suggest that we rename this prop to a more clear name, maybe `paletteLabelHeadingLevel`?